app/views/podcasts/_podcast.html.erb

```diff
@@ -24,6 +24,20 @@
         <h5 class="card-title fs-1 display-1"><%= @podcast.title %></h5>
         <h6 class="card-subtitle mb-2 text-body-secondary fs-6"><%= @podcast.created_at %></h6>
         <p class="card-text fs-5 display-2"><%= @podcast.description %></p>
+        <p class="card-text fs-5 display-2">User_id: <%= @podcast.user_id %></p>
+        <% if current_user && current_user.subscriptions.exists?(podcast: @podcast) %>
```

Comment:
let's show button for subscribing everytime also if user doesnt registered (In that case we should redirect them to login page)